spark/dl/src/test/scala/com/intel/analytics/bigdl/models/AlexNetSpec.scala

```diff
@@ -22,7 +22,7 @@ import com.intel.analytics.bigdl.nn._
 import com.intel.analytics.bigdl.numeric.NumericFloat
 import com.intel.analytics.bigdl.optim.SGD
 import com.intel.analytics.bigdl.tensor._
-import com.intel.analytics.bigdl.torch.TH
+import com.intel.analytics.bigdl.torch.{TH, TorchSpec}
 import com.intel.analytics.bigdl.utils.RandomGenerator._
 import com.intel.analytics.bigdl.utils.T
 import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
```

Comment:
You can/should remove these unused imports now too. That's more of the value of the cleanup.